.storybook/pages/StudentRefinement/StudentRefinement.module.css

```diff
@@ -82,7 +82,8 @@
  * The title communicates card organization.
  */
 .data-summary-card__title {
-  @mixin eds-theme-typography-overline-sm;
+  font: var(--eds-theme-typography-overline-sm);
+  text-transform: uppercase;
```

Comment:
There are a few examples of adding in `text-transform` to some cases, because `font` only supports CSS 2.1 values which doesn't include `all-caps` sadly